cmd/user-agent/src/pages/chapi/wallet/didmgmt/didManager.js

```diff
@@ -109,4 +109,18 @@ export class DIDManager extends KeyValueStore {
         return this.store(did, metadata)
     }
 
+    async saveDIDToSDS(friendlyName, signType, did){
```

Comment:
this implementation should be inside `saveDID()`

Reply:
Done!